### PR TITLE
changed vis in practice link to point to new, working website

### PR DESCRIPTION
**Pull request by bhentschel**
Dear Carlos,

**TL:DR**: I pointed the nav bar to the new ViP website. Could you please merge and deploy *ASAP*?

We have been trying to migrate the "old" URL visinpractice.org to my new provider. Albeit being standardized, this process has twice failed over the last two weeks. This resulted in significant downtime for visinpractice.org, something that we cannot afford with just over a week until the deadline. Therefore, I now hard-linked the nav bar to the new site, which at least gives us a valid entry point through ieeevis.org. We are working with both providers to get this settled asap. For now, this is the quickest hot fix I see.

Thanks in advance!

Bernd

### CODE REVIEW

**Comment by cscheid** [comment]
@jamesAhrens, this seems like a simple enough request that I'm going ahead and pushing to production before getting your ok.
